src/cli_plugin/cli.js

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash';
+import pkg from '../utils/packageJson';
+import Command from '../cli/Command';
```

Comment:
This separation in directory structure between the existing cli and the new plugin cli seems a bit awkward, and stuff like this really underscores that.  `cli_plugin` uses files from `cli` - why not just make them both part of the same module?